exercises/word-search/word_search_test.py

```diff
@@ -1,83 +1,53 @@
 import unittest
 
-from word_search import WordSearch, Point
+from word_search import search
 
 
 class WordSearchTests(unittest.TestCase):
 
-    @classmethod
-    def setUpClass(self):
-        puzzle = ('jefblpepre\n'
-                  'camdcimgtc\n'
-                  'oivokprjsm\n'
-                  'pbwasqroua\n'
-                  'rixilelhrs\n'
-                  'wolcqlirpc\n'
-                  'screeaumgr\n'
-                  'alxhpburyi\n'
-                  'jalaycalmp\n'
-                  'clojurermt')
-        self.example = WordSearch(puzzle)
+    puzzle = ['jefblpepre',
```

Comment:
Why did you change the input format? I mean it's only a `split()` away ... just would like to know the reasoning behind this.